dynamic-layers/multimedia-layer/recipes-multimedia/pipewire/pipewire_%.bbappend

```diff
@@ -6,7 +6,7 @@ SYSTEMD_AUTO_ENABLE:imx-nxp-bsp = "disable"
 
 DEPENDS:append:mx95-nxp-bsp = " libdrm"
 
-PACKAGECONFIG:remove:imx-nxp-bsp = "gstreamer"
+PACKAGECONFIG:remove:mx8mm-nxp-bsp = "gstreamer"
```

Comment:
For future reference, please add a comment mentioning why we are removing this option for this specific SoC.